docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "影子库压测"
+weight = 6
++++
+
+## Definition
+
+```sql
+CREATE SHADOW RULE shadowRuleDefinition [, shadowRuleDefinition] ... 
+
+ALTER SHADOW RULE shadowRuleDefinition [, shadowRuleDefinition] ... 
+
+ALTER SHADOW ALGORITHM shadowAlgorithm [, shadowAlgorithm] ...
+
+DROP SHADOW RULE ruleName [, ruleName] ...
+
+DROP SHADOW ALGORITHM algorithmName [, algorithmName] ...
+
+shadowRuleDefinition: ruleName(resourceMapping, shadowTableRule [, shadowTableRule] ...)
+
+resourceMapping: SOURCE=resourceName, SHADOW=resourceName
+
+shadowTableRule: tableName(shadowAlgorithm [, shadowAlgorithm] ...)
+
+shadowAlgorithm: ([algorithmName, ] TYPE(NAME=shadowAlgorithmType, PROPERTIES([algorithmProperties] ...)))
+
+algorithmProperties: algorithmProperty [, algorithmProperty] ... 
+
+algorithmProperty: key=value
+```
+
+- `Shadow` please refer to [Shadow](https://shardingsphere.apache.org/document/current/cn/features/shadow/)
+- `ruleName` Duplicate name cannot be created
```

Comment:
The order is not good, reference https://shardingsphere.apache.org/document/current/en/features/dist-sql/syntax/rdl/rdl-sharding-rule/